core/os/input_event.cpp

```diff
@@ -1204,7 +1271,7 @@ void InputEventTwistGesture::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_relative", "relative"), &InputEventTwistGesture::set_relative);
 	ClassDB::bind_method(D_METHOD("get_relative"), &InputEventTwistGesture::get_relative);
 
-	ADD_PROPERTY(PropertyInfo(Variant::REAL, "relative"), "set_relative", "get_relative");
+	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "relative"), "set_relative", "get_relative");
```

Comment:
These should probably be REAL

Reply:
REAL is not on the Variant class anymore.